PassleSync.Core/Services/ConfigService.cs

```diff
@@ -59,15 +59,25 @@ public IEnumerable<string> PassleShortcodes
         {
             get => PassleShortcodesString?.Split(',') ?? Enumerable.Empty<string>(); 
         }
-        public string PostPermalinkPrefix
+        public string PostPermalinkTemplate
         {
-            get => _keyValueService.GetValue("PassleSync.PostPermalinkPrefix");
-            set => _keyValueService.SetValue("PassleSync.PostPermalinkPrefix", value);
+            get => _keyValueService.GetValue("PassleSync.PostPermalinkTemplate");
+            set => _keyValueService.SetValue("PassleSync.PostPermalinkTemplate", value);
         }
-        public string AuthorPermalinkPrefix
+        public string PersonPermalinkTemplate
         {
-            get => _keyValueService.GetValue("PassleSync.PersonPermalinkPrefix");
-            set => _keyValueService.SetValue("PassleSync.PersonPermalinkPrefix", value);
+            get => _keyValueService.GetValue("PassleSync.PersonPermalinkTemplate");
+            set => _keyValueService.SetValue("PassleSync.PersonPermalinkTemplate", value);
+        }
+        public string PreviewPermalinkTemplate
+        {
+            get => _keyValueService.GetValue("PassleSync.PreviewPermalinkTemplate");
+            set => _keyValueService.SetValue("PassleSync.PreviewPermalinkTemplate", value);
+        }
+        public string SimulateRemoteHosting
+        {
+            get => _keyValueService.GetValue("PassleSync.SimulateRemoteHosting");
+            set => _keyValueService.SetValue("PassleSync.SimulateRemoteHosting", value);

```

Comment:
Can we do some conversion/parsing to and from boolean in this getter and setter so we don't have to do stuff like string comparisons later on

Reply:
Umbraco's key value service accepts strings as keys and values. I could try to do some conversions inside the get, set if it is better so when we get it, we look at its string value and return a boolean and when we set, we look at a boolean and set it as string.